client/web/src/enterprise/batches/detail/BatchChangeStatsCard.story.tsx

```diff
@@ -32,9 +37,11 @@ add('draft', () => (
             />
         )}
     </WebStory>
-))
+)
+
+draft.storyName = 'draft'
 
-add('open', () => (
+export const open: Story = () => (
```

Comment:
Let's stick to CamelCase notation for story exports.

```suggestion
export const Open: Story = () => (
```